server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -456,4 +477,146 @@ protected void onDelete() {
         // clean up all remote translog files
         translogTransferManager.delete();
     }
+
+    private class RemoteFsTranslogTransferListener implements TranslogTransferListener {
```

Comment:
please add @opensearch.internal.

Reply:
Addressed. Resolving this thread. Please reopen if any changes are needed.